src/phpDocumentor/Descriptor/Builder/Reflector/Tags/InvalidTagAssembler.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace phpDocumentor\Descriptor\Builder\Reflector\Tags;
+
+use phpDocumentor\Descriptor\Builder\Reflector\AssemblerAbstract;
+use phpDocumentor\Descriptor\TagDescriptor;
+use phpDocumentor\Reflection\DocBlock\Tags\InvalidTag;
+
+final class InvalidTagAssembler extends AssemblerAbstract
+{
+    /**
+     * @see $data
+     *
+     * @param InvalidTag $data
+     */
+    public function create($data) : TagDescriptor
+    {
+        $descriptor = new TagDescriptor($data->getName());
+        $descriptor->setDescription((string) $data);
+        $descriptor->getErrors()->add(
+            $data->getException()->getMessage()
```

Comment:
getErrors returns a Descriptor Collection; so one would expect this to contain descriptors instead of messages. If I look at the errors twig template; an object is expected with a severity, line, code and context (where the code is used to translate to the error).

It may be worth looking at phpDocumentor 2.9 and restore the Error object that we used there